apollo-router/src/plugins/headers.rs

```diff
@@ -57,16 +57,23 @@ enum Operation {
     Propagate(Propagate),
 }
 
+schemar_fn!(remove_named, String, "Remove a header given a header name");
+schemar_fn!(
+    remove_matching,
+    String,
+    "Remove a header given a regex matching header name"
```

Comment:
```suggestion
    "Remove a header given a regex matching against the header name"
```

Again though, happy to let these be a follow-up PR with editorial.